app/models/settings/general.rb

```diff
@@ -37,6 +37,10 @@ class General < Base
 
     setting :logo_svg, type: :string
 
+    setting :original_logo, type: :string
+    setting :resized_web_logo, type: :string
+    setting :resized_mobile_logo, type: :string
```

Comment:
W save three types of logo's:
- The original which we are not using at the moment but which we may use in the future when we replace logo_png.
- The resized_web_logo amends the size of the logo to fit in the header whilst maintaining the original aspect ratio. It allows us to serve smaller images to the browser
- - The resized_mobile_logo amends the size of the logo to fit in the header whilst maintaining the original aspect ratio. It allows us to serve smaller images to the browser